mne/utils/docs.py

```diff
@@ -4041,29 +4040,13 @@ def open_docs(kind=None, version=None):
     webbrowser.open_new_tab('https://mne.tools/%s/%s' % (version, kind))
 
 
-# Following deprecated class copied from scikit-learn
-
-# force show of DeprecationWarning even on python 2.7
-warnings.filterwarnings('always', category=DeprecationWarning, module='mne')
```

Comment:
Please revert, IIRC this is critical for helping our users see our warnings

Reply:
Python silences DeprecationWarnings when running in user code. 

It is therefore recommended to use FutureWarnings if we want to ensure that users see the warnings by default. 

We could keep this change here and change our Deprecation- to FutureWarnings

Reply:
Yes, I actually just learned a few days ago that FutureWarnings are for users and DeprecationWarnings are for downstream devs. But I didn't make the connection to this bit of code! @larsoner WDYT about switching our warning type?

Reply:
Works for me. Either revert this line for now and make a new PR for FutureWarning, or change to `FutureWarning` in this PR and then we can merge